medic/medic-appium.js

```diff
@@ -197,9 +214,10 @@ function parseArgs() {
     return options;
 }
 
-function getLocalCLI() {
-    if (util.isWindows()) {
-        return "cordova.bat";
+function getLocalCLI(appPath) {
+    if (util.isWindows() || !fs.existsSync(path.join(getFullAppPath(appPath), "./cordova"))) {
```

Comment:
Windows doesn't need the dot-slash to execute this .bat
So basically, on windows "cordova" command would execute .bat and if it's not there, will execute global cordova.